Contacts/People/Council/council.html

```diff
@@ -1,6 +1,8 @@
 ---
 title: The GAP Council
 layout: default
+toc: Contacts
+subtoc: People
```

Comment:
Hmmm... couldn't we also extract this data from the file paths?

Reply:
The directory structure is inconsistent. I started off with this hardcoded-ness, and then had the same thought (as in using the file paths), but quickly realised it wasn't going to work. Some files are in deeper folders, some are in the same folder and the nesting in the nav is not representative of that.

Reply:
Besides... I have already done nearly all of it now this way. 😂

Reply:
I would be happy to add comments to the template to make it clearer for anyone in the future. Just realised that is completely missing.

Reply:
OK, thanks for the explanation.

I think that in the future, I'd want this to be directly derived from the dir structure -- that we can't right now because it doesn't match IMHO indicates an issue with the website. But of course it's perfectly reasonable that you are focusing on getting this to work instead of wasting time trying to resolve that issue (which is of course much more complex, as one has to make decisions about where to put stuff; and stuff that we may want to remove soon anyway at that).

Reply:
I fully agree with the above.